translate/src/modules/history/components/HistoryTranslation.tsx

```diff
@@ -227,6 +233,17 @@ export function HistoryTranslationBase({
     !isReadOnlyEditor;
   let canComment = user.isAuthenticated;
 
+  const customDateFormat = (date: Date) => {
+    return new Intl.DateTimeFormat('en', {
+      year: 'numeric',
+      month: 'long',
+      day: 'numeric',
+      hour: 'numeric',
+      minute: 'numeric',
+      second: 'numeric',
```

Comment:
To be consistent with the format used for the review timestamp and more concise, let's use this:
```suggestion
      dateStyle: 'long',
      timeStyle: 'medium',
```